packages/traceability/issuer-api/src/pods/certification-request/handlers/get-all-certification-requests.handler.ts

```diff
@@ -16,6 +16,11 @@ export class GetAllCertificationRequestsHandler
     ) {}
 
     async execute({ query }: GetAllCertificationRequestsQuery): Promise<CertificationRequestDTO[]> {
-        return this.repository.find(query);
+        const q = query || {};
+        return this.repository.find({
+            ...(q.hasOwnProperty('approved') ? { approved: q.approved } : {}),
+            ...(q.owner ? { owner: q.owner } : {}),
+            ...(q.deviceIds?.length ? { deviceId: In(q.deviceIds) } : {})
+        });
```

Comment:
What's the reason for this change? It now only allows these 3 properties to be queried, and not others.

Reply:
We can add other fields later when we need it